src/start.php

```
@@ -1,7 +1,9 @@
 <?php
 require_once 'vendor/autoload.php';
 
+use Bcgov\NaadConnector\DestinationClient;
 use Bcgov\NaadConnector\NaadSocketClient;
 
-$connector = new NaadSocketClient($argv[1], $argv[2]);
+$destinationClient = new DestinationClient($argv[3], $argv[4], $argv[5]);
```

not a fan of this, i would try to parse out these, because what if there wasn't argv[5], although i suspect you are controlling this as well at worse maybe something like `$argv[3]??''` <- i think this is the php context

---

Yeah it doesn't look pretty but I'm not sure we want the app to even run if all of the arguments aren't provided because it won't do what it's intended to do. I think it's better that the app just fails to start instead of running silently with arguments we know won't allow it to work as expected.